### ISSUE TITLE
Thinks access key type is not SSH

### ISSUE THREAD

**Issue by mikkel1156** [ansible-semaphore/semaphore#855]
I'm running version v2.8.44 and it crashes (panics) thinking the access key type isn't SSH. I tried creating a new one, and even checked MySQL to see if the data was different somehow - it of course wasn't.

Here is the error I'm getting:
```
Server is running
time="2022-02-05T14:12:05Z" level=info msg="Task 62 added to queue"
time="2022-02-05T14:12:10Z" level=info msg="Set resource locker with TaskRunner 62"
time="2022-02-05T14:12:11Z" level=info msg="Stopped preparing TaskRunner 62"
time="2022-02-05T14:12:11Z" level=info msg="Release resource locker with TaskRunner 62"
panic: type must be ssh
goroutine 106 [running]:
github.com/ansible-semaphore/semaphore/db.AccessKey.GetSshCommand(0x3, 0xc0015f6538, 0x4, 0xc0015f653c, 0x4, 0xc0015f6540, 0x0, 0x0, 0x0, 0x0, ...)
	/go/src/github.com/ansible-semaphore/semaphore/db/AccessKey.go:157 +0x1de
github.com/ansible-semaphore/semaphore/lib.AnsiblePlaybook.makeCmd(0x4, 0x18b3310, 0xc001081080, 0x1, 0xc0015f6300, 0x4, 0x1, 0xc001b34d70, 0x4d, 0xc0015f6304, ...)
	/go/src/github.com/ansible-semaphore/semaphore/lib/AnsiblePlaybook.go:25 +0x145
github.com/ansible-semaphore/semaphore/lib.AnsiblePlaybook.GetHosts(0x4, 0x18b3310, 0xc001081080, 0x1, 0xc0015f6300, 0x4, 0x1, 0xc001b34d70, 0x4d, 0xc0015f6304, ...)
	/go/src/github.com/ansible-semaphore/semaphore/lib/AnsiblePlaybook.go:37 +0xd6
github.com/ansible-semaphore/semaphore/services/tasks.(*TaskRunner).listPlaybookHosts(0xc001081080, 0x0, 0x0)
	/go/src/github.com/ansible-semaphore/semaphore/services/tasks/runner.go:567 +0x150
github.com/ansible-semaphore/semaphore/services/tasks.(*TaskRunner).prepareRun(0xc001081080)
	/go/src/github.com/ansible-semaphore/semaphore/services/tasks/runner.go:257 +0x5e5
created by github.com/ansible-semaphore/semaphore/services/tasks.(*TaskPool).Run
	/go/src/github.com/ansible-semaphore/semaphore/services/tasks/pool.go:153 +0x6a5
```

I'm suspecting it's somehow not getting the right data from the database, since the [code](https://github.com/ansible-semaphore/semaphore/blob/develop/db/AccessKey.go#L157) it panics it on is so simple.

**Comment by fiftin** [comment]
Fixed in [v2.8.45](https://github.com/ansible-semaphore/semaphore/releases/tag/v2.8.45)

**Comment by mikkel1156** [comment]
Can confirm. Thanks for the prompt response!
